wlauto/resource_getters/standard.py

```diff
@@ -45,6 +45,9 @@ class PackageFileGetter(ResourceGetter):
     Looks for exactly one file with the specified extension in the owner's directory. If a version
     is specified on invocation of get, it will filter the discovered file based on that version.
     Versions are treated as case-insensitive.
+    If an ABI is specified in the resource request, then the getter will try to locate the file in
+    the ABI-specific folder in the form ``<root>/<file_extension>/<abi>/<filename>``. Where ``root``
+    is the base resource location e.g. ``~/.workload_automation/dependencies/<extension_name>``
```

Comment:
There are two ways of implementing this:
 1. Move the ABI check to `{Environment,Package}ApkGetter` and override the `get()` method, which means duplicating the code, or
 1. Add a method to provide `resource_dir` in `{Environment,Package}FileGetter` with a default implementation that returns the current value. The `ApkGetter` subclasses will override this to return the platform specific path, as would any other subclasses that might need this (good extensibility).